substrate-node/pallets/pallet-smart-contract/src/lib.rs

```diff
@@ -305,6 +315,20 @@ pub mod pallet {
         },
         SolutionProviderCreated(types::SolutionProvider<T::AccountId>),
         SolutionProviderApproved(u64, bool),
+        /// A Service contract is created
+        ServiceContractCreated {
+            service_contract_id: u64,
```

Comment:
Can you emit the full service contract object? Otherwise we cannot assemble it from graphql